Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandSeat.cpp

```diff
@@ -39,9 +39,10 @@ namespace WPE {
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(WaylandSeat);
 
-WaylandSeat::WaylandSeat(struct wl_seat* seat)
+WaylandSeat::WaylandSeat(struct wl_seat* seat, Function<void()>&& callback)
     : m_seat(seat)
     , m_keymap(adoptGRef(wpe_keymap_xkb_new()))
+    , m_capabilitiesChangedCallback(WTFMove(callback))
```

Comment:
I would add a explicit setter for the callback, in the constructor it's not obvious what the callback is for